internal/scanner/scanner.go

```diff
@@ -65,6 +66,12 @@ func (s *Scanner) ScanNamespace(namespace string) error {
 		fmt.Println("---------------------")
 	}
 
+	// TODO for next steps:
+	// Iterate through all auditableResources. Each item contains a list of resources and the policies that would need
+	// to evaluate them. You need to create the AdmissionReview request using the resource which is an unstructured.
+	// unstructured nested functions might help with that https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured
```

Comment:
unrelated, but I wonder about CRDs. I suppose in the future we could provide the CRD schemas when creating the dynamic client, and then it will all work with the unstructured resource?

Reply:
yes, I think that should work